integration/runner/runner.go

```diff
@@ -79,7 +104,7 @@ func PushAndRunTests(host, testDir string) error {
 	portStr := strconv.Itoa(*port)
 	errChan := make(chan error)
 	go func() {
-		args = common.GetGCComputeArgs("ssh", host, "--", fmt.Sprintf("sudo %s --port %s --logtostderr", path.Join(testDir, cadvisorBinary), portStr))
+		args = common.GetGCComputeArgs("ssh", host, "--", fmt.Sprintf("sudo %s --port %s --logtostderr &> %s/log.txt", path.Join(testDir, cadvisorBinary), portStr, testDir))
```

Comment:
I believe this will only redirect stdout, and not stderr.  Is that your intent?  Could we use --alsologtostderr and then set the log file to <>/log.txt so that the error output still appears in Jenkins?